src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorConfigurationService.cs

```diff
@@ -49,16 +53,22 @@ public async override Task<RazorLSPOptions> GetLatestOptionsAsync()
                 };
 
                 var result = await _server.Client.SendRequest<ConfigurationParams, object[]>("workspace/configuration", request);
-                if (result == null || result.Length < 1 || result[0] == null)
+                if (result == null || result.Length < 2 || result[0] == null)
```

Comment:
What makes 2 the expected length, and as a corollary, why was it one before?

Reply:
The spec says the result should be the same length as the number of `ConfigurationItem`s we pass in. I'll leave a comment for future us.